engine/src/main/java/org/hibernate/validator/internal/engine/path/NodeImpl.java

```diff
@@ -56,6 +56,7 @@
 	public static final String MAP_VALUE_NODE_NAME = "<map value>";
 
 	private final String name;
+	private final String resolvedName;
```

Comment:
Do we need to keep both names resolved/original ones ? I was thinking that we could keep just one name. Then we also wouldn't need the `getResolvedPropertyPath()` method in `ConstraintViolationImpl`

Reply:
This is done, please review and resolve if the change is ok.